<<<<<<< HEAD
ERROR - 2022-02-11 19:39:50 --> Error connecting to the database: No connection could be made because the target machine actively refused it.

CRITICAL - 2022-02-11 19:39:50 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

#0 C:\xampp\htdocs\wholesales\client-report\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\wholesales\client-report\system\Database\BaseBuilder.php(1576): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\wholesales\client-report\system\Model.php(723): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 C:\xampp\htdocs\wholesales\client-report\app\Controllers\Auth.php(38): CodeIgniter\Model->__call('getWhere', Array)
#4 C:\xampp\htdocs\wholesales\client-report\system\CodeIgniter.php(824): App\Controllers\Auth->loginProses()
#5 C:\xampp\htdocs\wholesales\client-report\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\xampp\htdocs\wholesales\client-report\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\wholesales\client-report\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\wholesales\client-report\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2022-02-11 19:39:51 --> Uncaught ErrorException: print_r(): Property access is not allowed yet in C:\xampp\htdocs\wholesales\client-report\app\Views\errors\html\error_exception.php:96
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'print_r(): Prop...', 'C:\\xampp\\htdocs...', 96, Array)
#1 C:\xampp\htdocs\wholesales\client-report\app\Views\errors\html\error_exception.php(96): print_r(Object(App\Controllers\Auth), true)
#2 C:\xampp\htdocs\wholesales\client-report\system\Debug\Exceptions.php(229): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\wholesales\client-report\system\Debug\Exceptions.php(232): CodeIgniter\Debug\Exceptions->CodeIgniter\Debug\{closure}()
#4 C:\xampp\htdocs\wholesales\client-report\system\Debug\Exceptions.php(116): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Database\Exceptions\DatabaseException), 500)
#5 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DatabaseException))
#6 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
=======
ERROR - 2022-02-11 00:45:01 --> Error connecting to the database: No such file or directory
CRITICAL - 2022-02-11 00:45:01 --> Unable to connect to the database.
Main connection [MySQLi]: No such file or directory
#0 /Applications/MAMP/htdocs/smartfba/client-report/system/Database/BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 /Applications/MAMP/htdocs/smartfba/client-report/system/Database/BaseBuilder.php(1576): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /Applications/MAMP/htdocs/smartfba/client-report/system/Model.php(723): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 /Applications/MAMP/htdocs/smartfba/client-report/app/Controllers/Auth.php(38): CodeIgniter\Model->__call('getWhere', Array)
#4 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(824): App\Controllers\Auth->loginProses()
#5 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /Applications/MAMP/htdocs/smartfba/client-report/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 /Applications/MAMP/htdocs/smartfba/client-report/system/Commands/Server/rewrite.php(43): require_once('/Applications/M...')
#9 {main}
ERROR - 2022-02-11 00:46:40 --> Error connecting to the database: No such file or directory
CRITICAL - 2022-02-11 00:46:40 --> Unable to connect to the database.
Main connection [MySQLi]: No such file or directory
#0 /Applications/MAMP/htdocs/smartfba/client-report/system/Database/BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 /Applications/MAMP/htdocs/smartfba/client-report/system/Database/BaseBuilder.php(1576): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /Applications/MAMP/htdocs/smartfba/client-report/system/Model.php(723): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 /Applications/MAMP/htdocs/smartfba/client-report/app/Controllers/Auth.php(38): CodeIgniter\Model->__call('getWhere', Array)
#4 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(824): App\Controllers\Auth->loginProses()
#5 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /Applications/MAMP/htdocs/smartfba/client-report/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 /Applications/MAMP/htdocs/smartfba/client-report/system/Commands/Server/rewrite.php(43): require_once('/Applications/M...')
#9 {main}
ERROR - 2022-02-11 00:47:04 --> Error connecting to the database: No such file or directory
CRITICAL - 2022-02-11 00:47:04 --> Unable to connect to the database.
Main connection [MySQLi]: No such file or directory
#0 /Applications/MAMP/htdocs/smartfba/client-report/system/Database/BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 /Applications/MAMP/htdocs/smartfba/client-report/system/Database/BaseBuilder.php(1576): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /Applications/MAMP/htdocs/smartfba/client-report/system/Model.php(723): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 /Applications/MAMP/htdocs/smartfba/client-report/app/Controllers/Auth.php(38): CodeIgniter\Model->__call('getWhere', Array)
#4 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(824): App\Controllers\Auth->loginProses()
#5 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /Applications/MAMP/htdocs/smartfba/client-report/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 /Applications/MAMP/htdocs/smartfba/client-report/system/Commands/Server/rewrite.php(43): require_once('/Applications/M...')
#9 {main}
ERROR - 2022-02-11 00:49:15 --> Error connecting to the database: No such file or directory
CRITICAL - 2022-02-11 00:49:15 --> Unable to connect to the database.
Main connection [MySQLi]: No such file or directory
#0 /Applications/MAMP/htdocs/smartfba/client-report/system/Database/BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 /Applications/MAMP/htdocs/smartfba/client-report/system/Database/BaseBuilder.php(1576): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /Applications/MAMP/htdocs/smartfba/client-report/system/Model.php(723): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 /Applications/MAMP/htdocs/smartfba/client-report/app/Controllers/Auth.php(38): CodeIgniter\Model->__call('getWhere', Array)
#4 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(824): App\Controllers\Auth->loginProses()
#5 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /Applications/MAMP/htdocs/smartfba/client-report/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 /Applications/MAMP/htdocs/smartfba/client-report/system/Commands/Server/rewrite.php(43): require_once('/Applications/M...')
#9 {main}
ERROR - 2022-02-11 00:49:47 --> Error connecting to the database: No such file or directory
CRITICAL - 2022-02-11 00:49:47 --> Unable to connect to the database.
Main connection [MySQLi]: No such file or directory
#0 /Applications/MAMP/htdocs/smartfba/client-report/system/Database/BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 /Applications/MAMP/htdocs/smartfba/client-report/system/Database/BaseBuilder.php(1576): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /Applications/MAMP/htdocs/smartfba/client-report/system/Model.php(723): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 /Applications/MAMP/htdocs/smartfba/client-report/app/Controllers/Auth.php(38): CodeIgniter\Model->__call('getWhere', Array)
#4 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(824): App\Controllers\Auth->loginProses()
#5 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /Applications/MAMP/htdocs/smartfba/client-report/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 /Applications/MAMP/htdocs/smartfba/client-report/system/Commands/Server/rewrite.php(43): require_once('/Applications/M...')
#9 {main}
ERROR - 2022-02-11 00:50:49 --> Error connecting to the database: No such file or directory
CRITICAL - 2022-02-11 00:50:49 --> Unable to connect to the database.
Main connection [MySQLi]: No such file or directory
#0 /Applications/MAMP/htdocs/smartfba/client-report/system/Database/BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 /Applications/MAMP/htdocs/smartfba/client-report/system/Database/BaseBuilder.php(1576): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /Applications/MAMP/htdocs/smartfba/client-report/system/Model.php(723): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 /Applications/MAMP/htdocs/smartfba/client-report/app/Controllers/Auth.php(38): CodeIgniter\Model->__call('getWhere', Array)
#4 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(824): App\Controllers\Auth->loginProses()
#5 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /Applications/MAMP/htdocs/smartfba/client-report/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 /Applications/MAMP/htdocs/smartfba/client-report/system/Commands/Server/rewrite.php(43): require_once('/Applications/M...')
#9 {main}
ERROR - 2022-02-11 00:52:49 --> Expression #1 of SELECT list is not in GROUP BY clause and contains nonaggregated column 'rms.investments.date' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by
CRITICAL - 2022-02-11 00:52:49 --> Expression #1 of SELECT list is not in GROUP BY clause and contains nonaggregated column 'rms.investments.date' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by
#0 /Applications/MAMP/htdocs/smartfba/client-report/system/Database/MySQLi/Connection.php(292): mysqli->query('select month, s...', 0)
#1 /Applications/MAMP/htdocs/smartfba/client-report/system/Database/BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('select month, s...')
#2 /Applications/MAMP/htdocs/smartfba/client-report/system/Database/BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('select month, s...')
#3 /Applications/MAMP/htdocs/smartfba/client-report/app/Models/ReportModel.php(170): CodeIgniter\Database\BaseConnection->query('select month, s...')
#4 /Applications/MAMP/htdocs/smartfba/client-report/app/Controllers/Admin/Reports.php(51): App\Models\ReportModel->finSummary()
#5 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(824): App\Controllers\Admin\Reports->index()
#6 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Reports))
#7 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /Applications/MAMP/htdocs/smartfba/client-report/public/index.php(37): CodeIgniter\CodeIgniter->run()
#9 /Applications/MAMP/htdocs/smartfba/client-report/system/Commands/Server/rewrite.php(43): require_once('/Applications/M...')
#10 {main}
CRITICAL - 2022-02-11 01:34:21 --> Call to undefined method App\Models\InvestmentModel::getTopReadyToAssign
#0 /Applications/MAMP/htdocs/smartfba/client-report/app/Controllers/Admin/Reports.php(968): CodeIgniter\Model->__call('getTopReadyToAs...', Array)
#1 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(824): App\Controllers\Admin\Reports->getTopReadyToAssign()
#2 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Reports))
#3 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 /Applications/MAMP/htdocs/smartfba/client-report/public/index.php(37): CodeIgniter\CodeIgniter->run()
#5 /Applications/MAMP/htdocs/smartfba/client-report/system/Commands/Server/rewrite.php(43): require_once('/Applications/M...')
#6 {main}
CRITICAL - 2022-02-11 01:34:39 --> Call to undefined method App\Models\InvestmentModel::getTopReadyToAssign
#0 /Applications/MAMP/htdocs/smartfba/client-report/app/Controllers/Admin/Reports.php(968): CodeIgniter\Model->__call('getTopReadyToAs...', Array)
#1 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(824): App\Controllers\Admin\Reports->getTopReadyToAssign()
#2 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Reports))
#3 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 /Applications/MAMP/htdocs/smartfba/client-report/public/index.php(37): CodeIgniter\CodeIgniter->run()
#5 /Applications/MAMP/htdocs/smartfba/client-report/system/Commands/Server/rewrite.php(43): require_once('/Applications/M...')
#6 {main}
CRITICAL - 2022-02-11 01:37:01 --> Call to undefined method App\Models\InvestmentModel::getTopReadyToAssign
#0 /Applications/MAMP/htdocs/smartfba/client-report/app/Controllers/Admin/Reports.php(968): CodeIgniter\Model->__call('getTopReadyToAs...', Array)
#1 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(824): App\Controllers\Admin\Reports->getTopReadyToAssign()
#2 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Reports))
#3 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 /Applications/MAMP/htdocs/smartfba/client-report/public/index.php(37): CodeIgniter\CodeIgniter->run()
#5 /Applications/MAMP/htdocs/smartfba/client-report/system/Commands/Server/rewrite.php(43): require_once('/Applications/M...')
#6 {main}
ERROR - 2022-02-11 02:10:49 --> You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ' UPPER(category) as name FROM `assign_report_details` GROUP BY category' at line 1
CRITICAL - 2022-02-11 02:10:49 --> You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ' UPPER(category) as name FROM `assign_report_details` GROUP BY category' at line 1
#0 /Applications/MAMP/htdocs/smartfba/client-report/system/Database/MySQLi/Connection.php(292): mysqli->query('SELECT COUNT(ca...', 0)
#1 /Applications/MAMP/htdocs/smartfba/client-report/system/Database/BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(ca...')
#2 /Applications/MAMP/htdocs/smartfba/client-report/system/Database/BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(ca...')
#3 /Applications/MAMP/htdocs/smartfba/client-report/app/Models/AssignReportModel.php(108): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(ca...')
#4 /Applications/MAMP/htdocs/smartfba/client-report/app/Controllers/Admin/Reports.php(973): App\Models\AssignReportModel->getTotalItemByCat()
#5 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(824): App\Controllers\Admin\Reports->getTotalItemByCat()
#6 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Reports))
#7 /Applications/MAMP/htdocs/smartfba/client-report/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /Applications/MAMP/htdocs/smartfba/client-report/public/index.php(37): CodeIgniter\CodeIgniter->run()
#9 /Applications/MAMP/htdocs/smartfba/client-report/system/Commands/Server/rewrite.php(43): require_once('/Applications/M...')
#10 {main}
>>>>>>> 78ec1d315f2fbcb6ecd76f0bbf506bbace29c9fd
ERROR - 2022-02-11 19:41:18 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-02-11 19:41:18 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\wholesales\client-report\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\wholesales\client-report\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\wholesales\client-report\system\Model.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\wholesales\client-report\system\BaseModel.php(523): CodeIgniter\Model->doFind(true, '3')
#4 C:\xampp\htdocs\wholesales\client-report\app\Controllers\Admin\Reports.php(46): CodeIgniter\BaseModel->find('3')
#5 C:\xampp\htdocs\wholesales\client-report\system\CodeIgniter.php(824): App\Controllers\Admin\Reports->index()
#6 C:\xampp\htdocs\wholesales\client-report\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Reports))
#7 C:\xampp\htdocs\wholesales\client-report\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\wholesales\client-report\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\wholesales\client-report\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-02-11 19:41:18 --> Uncaught ErrorException: print_r(): Property access is not allowed yet in C:\xampp\htdocs\wholesales\client-report\app\Views\errors\html\error_exception.php:96
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'print_r(): Prop...', 'C:\\xampp\\htdocs...', 96, Array)
#1 C:\xampp\htdocs\wholesales\client-report\app\Views\errors\html\error_exception.php(96): print_r(Object(App\Controllers\Admin\Reports), true)
#2 C:\xampp\htdocs\wholesales\client-report\system\Debug\Exceptions.php(229): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\wholesales\client-report\system\Debug\Exceptions.php(232): CodeIgniter\Debug\Exceptions->CodeIgniter\Debug\{closure}()
#4 C:\xampp\htdocs\wholesales\client-report\system\Debug\Exceptions.php(116): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Database\Exceptions\DatabaseException), 500)
#5 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DatabaseException))
#6 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
ERROR - 2022-02-11 19:41:53 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-02-11 19:41:53 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\wholesales\client-report\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\wholesales\client-report\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\wholesales\client-report\system\Model.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\wholesales\client-report\system\BaseModel.php(523): CodeIgniter\Model->doFind(true, '3')
#4 C:\xampp\htdocs\wholesales\client-report\app\Controllers\Admin\Reports.php(46): CodeIgniter\BaseModel->find('3')
#5 C:\xampp\htdocs\wholesales\client-report\system\CodeIgniter.php(824): App\Controllers\Admin\Reports->index()
#6 C:\xampp\htdocs\wholesales\client-report\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Reports))
#7 C:\xampp\htdocs\wholesales\client-report\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\wholesales\client-report\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\wholesales\client-report\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-02-11 19:41:53 --> Uncaught ErrorException: print_r(): Property access is not allowed yet in C:\xampp\htdocs\wholesales\client-report\app\Views\errors\html\error_exception.php:96
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'print_r(): Prop...', 'C:\\xampp\\htdocs...', 96, Array)
#1 C:\xampp\htdocs\wholesales\client-report\app\Views\errors\html\error_exception.php(96): print_r(Object(App\Controllers\Admin\Reports), true)
#2 C:\xampp\htdocs\wholesales\client-report\system\Debug\Exceptions.php(229): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\wholesales\client-report\system\Debug\Exceptions.php(232): CodeIgniter\Debug\Exceptions->CodeIgniter\Debug\{closure}()
#4 C:\xampp\htdocs\wholesales\client-report\system\Debug\Exceptions.php(116): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Database\Exceptions\DatabaseException), 500)
#5 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DatabaseException))
#6 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
